### ISSUE TITLE
Edge (TGL) triggers segfault on Linux

### ISSUE THREAD

**Issue by softtest02** [joncampbell123/dosbox-x#1254]
Attempt to run the game. The game's audio driver will load and then Dosbox-X will segfault before the game can start.

I have only been able to reproduce the issue on Linux. On my Windows install Dosbox-X doesn't segfault.

conf: https://pastebin.com/raw/JdKBwM5L

**Comment by joncampbell123** [comment]
Try running without the MPU-401 enabled.

The ALSA MIDI code has never been tested on my end, my tests use FluidSynth.

**Comment by softtest02** [comment]
Yes, it runs when I set mpu401              = none. It's probably an issue with the ALSA MIDI code then.

**Comment by joncampbell123** [comment]
I'll need help with that then, I do not have a working setup and external MIDI device necessary to test ALSA MIDI output.
